kong/vault_service_test.go

```diff
@@ -0,0 +1,207 @@
+package kong
+
+import (
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestVaultsService(t *testing.T) {
+	RunWhenEnterprise(t, ">=3.0.0", RequiredFeatures{})
+	assert := assert.New(t)
+	require := require.New(t)
+
+	client, err := NewTestClient(nil, nil)
+	assert.NoError(err)
+	assert.NotNil(client)
```

Comment:
Ditto here (and elsewhere)

```suggestion
	require.NotNil(client)
```